app/src/main/java/com/nervousfish/nervousfish/modules/cryptography/EncryptorAdapter.java

```diff
@@ -1,17 +1,45 @@
 package com.nervousfish.nervousfish.modules.cryptography;
 
+import android.util.Base64;
+
+import com.nervousfish.nervousfish.exceptions.EncryptionException;
 import com.nervousfish.nervousfish.service_locator.IServiceLocator;
 import com.nervousfish.nervousfish.service_locator.ModuleWrapper;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+import java.util.Random;
+
+import javax.crypto.BadPaddingException;
+import javax.crypto.Cipher;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.NoSuchPaddingException;
+import javax.crypto.SecretKey;
+import javax.crypto.SecretKeyFactory;
+import javax.crypto.spec.PBEKeySpec;
+import javax.crypto.spec.PBEParameterSpec;
+
 /**
  * An adapter to the default Java class for encrypting messages
  */
 public final class EncryptorAdapter implements IEncryptor {
     private static final long serialVersionUID = 5930930748980177440L;
     private static final Logger LOGGER = LoggerFactory.getLogger("EncryptorAdapter");
+    private static final String PBE_WITH_MD5_AND_DES = "PBEWithMD5AndDES";
+    private static final String UTF_8_NO_LONGER_SUPPORTED = "UTF-8 is no longer an encoding algorithm";
+    private static final String CANNOT_HAPPEN_UTF_8 = "Cannot happen, no encodign algorithm like UTF-8";
```

Comment:
A few points:
- There are typos in these strings.
- You're not properly using them, for example on [line 84](https://github.com/ericcornelissen/NervousFish/pull/252/files/65ae0f3a84342b70782e9ba8a6ae88a5f37d105d#diff-4a0ae8e2bf0ce74e2c03082d5ace1a3dR84) you have hardcoded the `CANNOT_HAPPEN_UTF_8` text.
- You're not consistently using them. For example, you don't have a string for the SHA-256 error.
- If you have to rewrite it so often you might consider refactoring so you have less duplicate code. E.g. for the UTF error:
```java
private String getKeyAsString(final byte[] key) {
     try {
         return new String(secretString, UTF_8);
     } catch (final UnsupportedEncodingException e) {
         LOGGER.error("the message", e);
         throw new EncryptionException("the message", e);
     }
}
```